documentation/dependencies/pip.dependencies

```diff
@@ -1 +1 @@
-sphinx-book-theme
+sphinx-book-theme==0.2.0
```

Comment:
I'm ok with this change, but I think this fix is also a workaround. In the commit message, you said that this issue was caused due to a change in how sphinx-book gets the theme path; in this sense, we should fix our doc to work with the latest version of this pip project.

Reply:
Thanks for your review comments. I make a new PR using a better way. https://github.com/kworkflow/kworkflow/pull/603